app.py

```diff
@@ -55,6 +87,58 @@ def choose_game_type_handler(handler_input: HandlerInput) -> Response:
     return handler_input.response_builder.response
 
 
+@sb.request_handler(can_handle_func=lambda input:
+                    is_currently_playing(input) and
+                    is_intent_name('NumberGuessIntent')(input))
```

Comment:
Don't really like 'NumberGuess', let's say 'AnswerIntent' or something like that?